temporalio/workflow_service.py

```diff
@@ -26,6 +27,10 @@
     "WorkflowServiceResponse", bound=google.protobuf.message.Message
 )
 
+logger = logging.getLogger(__name__)
+
+LOG_PROTOS = False
```

Comment:
Document this?

Reply:
I'd just mention why this is here and why you'd want to turn it on.

Reply:
Will do. I do find myself turning it on so often in tests I am considering making it externally enableable.

Reply:
Documented